pages/applications/index.vue

```diff
@@ -1,98 +1,123 @@
 <template>
-    <div>
-<div class="container">
-  <h1 class="page-title">Grants</h1>
-  <p class="page-description">The Vatterott Foundation funds organizations working in its Focus Areas, based on specific goals and strategies outlined by the programs. Grants are awarded solely for charitable purposes. Trustees review applications quarterly. The Foundation accepts applications on a rolling/ ongoing basis. Deadlines for organizations to be are considered at the quarterly meetings: February 15, May 15, August 15, and November 15.</p>
-  <section>
-    <h2 class="page-section-header">Determine Eligibility</h2>
-    <p class="section-description">In order to be considered for a grant your organization and request must meet the following requirements:
-      <ol>
-        <li>
-          <h4 class="grant-requirement">Work in one of our four Focus Areas</h4>
-          <ul>
-            <li>Amelioration of human poverty, sickness, and distress</li>
-            <li>Advancement of education, science, and learning,</li>
-            <li>Promotion of public welfare</li>
-            <li>Advancement of religion</li>
-          </ul>
-        </li>
-        <li>
-          <h4 class="grant-requirement">Be a 501(c)(3) Non-Profit Accredited Organization</h4>
-        </li>
-        <li>
-          <h4 class="grant-requirement">Located in the Greater St. Louis Metropolitan Area</h4>
-        </li>
-        <li>
-          <h4 class="grant-requirement">Operate with Respect to Life policies and practices</h4>
-        </li>
-        <li>
-          <h4 class="grant-requirement">Your proposal is one of the following Funding Types:</h4>
-          <ul>
-            <li>General Operation</li>
-            <li>Capital Campaigns</li>
-            <li>Specific Program Support</li>
-            <li><strong>Note:</strong> Grants to individuals are <em>not</em> available</li>
-          </ul>
-        </li>
-      </ol>
-    </p>
-  </section>
-  <section>
-    <h2 class="page-section-header">Submit New Application</h2>
-    <p class="page-description">
-      If you feel that your organization or project is a good match with the Vatterott Foundation’s funding priorities, you may apply for a grant by downloading and submitting the following documents. Please <a href="mailto:info@vatterottfoundation.org">email</a> us if you have any questions.
-    </p>
+  <div>
+    <div class="container">
+      <h1 class="page-title">{{ title }}</h1>
+      <p class="page-description">
+        {{ subtitle }}
+      </p>
+      <div v-html="html" />
+      <div>
+        <h2>{{ appdescriptions.header2 }}</h2>
+
+        <div class="page-description" style="text-align: center;">
+           {{ appdescriptions.goodmatchA }}
+        <a href="mailto:info@vatterottfoundation.org">{{ appdescriptions.emailtextlink }}</a>
+            {{ appdescriptions.goodmatchB }}
+        </div>
+      </div>
```

Comment:
I think this `appdescriptions` section should also be part of the page markdown

Reply:
I've been trying to figure out how to put that text in markdown while including the styles and the link that are necessary. I'm stuck like the little blue truck.

Reply:
I think I finished this... on to the next